docs/hugo/content/introduction/multitenant-deployment.md

```diff
@@ -0,0 +1,78 @@
+# Deploying ASO v2 in multi-tenant mode
+
+The current release version of the ASO deployment YAML installs the operator in single-tenant mode:
+it's deployed in the `azureserviceoperator-system` namespace with one set of Azure credentials, and manages resources in any namespace in the cluster.
+That single operator deployment handles webhooks fired when Azure resources are changed.
+
+ASO can also be deployed in a multi-tenant configuration that enables using separate credentials to manage resources in different Kubernetes namespaces.
+
+Running the operator in multi-tenant mode requires one deployment to handle webhooks (since webhook configurations are cluster-level resources) and then a deployment for each tenant, each with its own credentials and set of namespaces that it watches for Azure resources.
+To deploy the operator in multi-tenant mode the release YAML has been split into two parts:
+
+1. **Cluster-wide resources**:
+   * Custom resource definitions for the Azure resources.
+
+   * Cluster roles for managing those resources.
+
+   * The `azureserviceoperator-system` namespace containing the deployment and service to handle ASO webhooks.
+     The webhook service is a deployment of the ASO operator image, but configured to run in webhook-only mode.
+     It won't try to reconcile Azure resources with ARM, and so doesn't need Azure credentials.
+
+   * Webhook configuration referring to that service.
+
+
+2. **Per-tenant resources**:
+   * A namespace containing the deployment to run the tenant operator, configured for watchers-only mode.
+
+   * The `aso-controller-settings` secret defining the Azure credentials that should be used, and the set of namespaces that this operator will watch for Azure resources.
+
+   * A cluster role binding enabling the per-tenant operator's service account to manage the Azure resources.
+
+## Example files
+Examples of the deployment YAML files are available on the release page for ASO v2 releases from [v2.0.0-alpha.5](https://github.com/Azure/azure-service-operator/releases/tag/v2.0.0-alpha.5).
+The cluster-wide file `multitenant-cluster_v2.0.0-alpha.5.yaml` can be used as-is (the webhook deployment namespace is fixed as `azureserviceoperator-system`),
+but the namespaces and cluster role binding in the per-tenant file `multitenant-tenant_v2.0.0-alpha.5.yaml` will need to be updated from `tenant1` to the desired name for each tenant.
```

Comment:
Trying to make is more apparent that a separate file is needed for each tenant.

```suggestion
The cluster-wide file `multitenant-cluster_v2.0.0-alpha.5.yaml` can be used as-is (the webhook deployment namespace is fixed as `azureserviceoperator-system`),
but the namespaces and cluster role binding in the per-tenant file `multitenant-tenant_v2.0.0-alpha.5.yaml` will need to be customized in each tenant's configuration file from `tenant1` to the desired name for that tenant.
```

Reply:
Except I used `YAML file` rather than `configuration file` because I don't want it to be confused with the configuration in the secret below.